renovate.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": [
-    "config:base",
+    "mergeConfidence:all-badges",
```

Comment:
Why is this needed? 
I understood, this would only need to be enabled for self-hosting renovate: https://docs.renovatebot.com/merge-confidence/#enabling-and-disabling

Reply:
The massaged config contains the values that are in fact used.
One way is to make the config transparent. e.g. when i run renovate locally to test renovate, then this preset is not set.
But in both cases i would want the same output.
But it is not important, i can also drop this commit